src/model.js

```diff
@@ -31,7 +31,7 @@ const defineRelationship = (object, field, attribute) => {
   Object.defineProperty(object, field, {
     get() {
       const find = ({ id } = {}) => Registry.get(resource).peek(id);
-      const fetch = (isArray ? (data = []) => data.map(find) : find);
+      const fetch = (isArray ? (data = []) => data.map(find).filter(Boolean) : find);
```

Comment:
`filter(Boolean)`, huh? clever. i did not know that was a thing you could do.

Reply:
I'd be lying if I said I didn't google `javascript compact`